docs/glossary.rst

```diff
@@ -0,0 +1,72 @@
+.. _glossary:
+
+Glossary
+========
+
+The key words *"MUST"*, *"MUST NOT"*, *"REQUIRED"*, *"SHALL"*, *"SHALL NOT"*, *"SHOULD"*,
+*"SHOULD NOT"*, *"RECOMMENDED"*,  *"MAY"*, and *"OPTIONAL"* in this document are to be
+interpreted as described in :rfc:`2119`.
+
+.. glossary::
+    :sorted:
+
+    Client
+        An application which connects to the MetricQ network.
+        It is uniquely identified on the network by its :term:`Token`.
+        Clients may produce and consume data (see :term:`Source`, :term:`Sink`) or perform management task (see :term:`Manager`).
+
+    Sink
+        A :term:`Client` which receives :term:`Data Points <Data Point>` for one or several :term:`Metrics<Metric>` provided by other clients on the network.
+
+        Use :class:`metricq.Sink` to build your own Sinks;
+        see :ref:`sink-how-to` for a quick how-to.
+
+    Source
+        A :term:`Client` which produces :term:`Data Points <Data Point>` for one or several :term:`Metrics<Metric>` which are then available for consumption by other Clients on the network.
+
+    Manager
+        A :term:`Client` responsible for managing RPC requests on the network.
+
+    Data Point
+        A *time-value pair*, describing a measurement of a measurand at a specific point in time.
+
+    Metric
+        A *named* series of *time-value pairs*, called :term:`Data Points<Data Point>`.
+        Metrics are at the core of the MetricQ data exchange.
+        They describe how one specific *measurand* (e.g. the temperature in room E27 in ℃, CPU utilization on system ``hal-9000`` in %) evolves over time.
+        Clients must ensure the following properties for metrics they produce:
+
+        Metric names must be *unique* on the network
+            For any metric :math:`m`, at any point in time, there shall be *at most one* :term:`Source` online which produces data points for :math:`m`.
+            A single source may of course produce values for many distinct metrics.
+
+        Any valid Metric must be *strongly chronologically increasing*:
```

Comment:
@tilsche Do we really require **strict** monotonicity in the temporal dimension?

Reply:
As in the `metricq-db-hta` implementation, **yes**.
As in it hurts my head thinking about the case, also, **yes**